Build failure with persistent 2.9

```
> /tmp/stackage-build13$ stack unpack esqueleto-2.6.1@sha256:660d5fa65148ca59370b7f78b5ec1c61150fa789176873323ca9b7661718ffb7
Unpacked esqueleto-2.6.1 to /tmp/stackage-build13/esqueleto-2.6.1/
> /tmp/stackage-build13/esqueleto-2.6.1$ ghc -clear-package-db -global-package-db -package-db=/var/stackage/work/builds/nightly/pkgdb Setup
[1 of 1] Compiling Main             ( Setup.hs, Setup.o )
Linking Setup ...
> /tmp/stackage-build13/esqueleto-2.6.1$ ./Setup configure --package-db=clear --package-db=global --package-db=/var/stackage/work/builds/nightly/pkgdb --libdir=/var/stackage/work/builds/nightly/lib --bindir=/var/stackage/work/builds/nightly/bin --datadir=/var/stackage/work/builds/nightly/share --libexecdir=/var/stackage/work/builds/nightly/libexec --sysconfdir=/var/stackage/work/builds/nightly/etc --docdir=/var/stackage/work/builds/nightly/doc/esqueleto-2.6.1 --htmldir=/var/stackage/work/builds/nightly/doc/esqueleto-2.6.1 --haddockdir=/var/stackage/work/builds/nightly/doc/esqueleto-2.6.1
Configuring esqueleto-2.6.1...
> /tmp/stackage-build13/esqueleto-2.6.1$ ghc -clear-package-db -global-package-db -package-db=/var/stackage/work/builds/nightly/pkgdb Setup
> /tmp/stackage-build13/esqueleto-2.6.1$ ./Setup build
Preprocessing library for esqueleto-2.6.1..
Building library for esqueleto-2.6.1..
[1 of 8] Compiling Database.Esqueleto.Internal.PersistentImport ( src/Database/Esqueleto/Internal/PersistentImport.hs, dist/build/Database/Esqueleto/Internal/PersistentImport.o )

src/Database/Esqueleto/Internal/PersistentImport.hs:56:3: error:
    Not in scope: ‘executeQQ’
   |
56 |   executeQQ,
   |   ^^^^^^^^^

src/Database/Esqueleto/Internal/PersistentImport.hs:57:3: error:
    Not in scope: ‘sqlQQ’
   |
57 |   sqlQQ,
   |   ^^^^^
```

--- a/changelog.md
+++ b/changelog.md
@@ -1,6 +1,12 @@
 Unreleased
 ========
 
+2.6.2
+=======
+
+- @parsonsmatt
+  - [#117](https://github.com/bitemyapp/esqueleto/pull/117): Removed `sqlQQ` and `executeQQ` functions from export, fixing doc build and building with `persistent` >= 2.9
+
 2.6.1
 =======
 

--- a/changelog.md
+++ b/changelog.md
@@ -1,7 +1,7 @@
 Unreleased
 ========
 
-2.6.2
+2.7.0
 =======
 
 - @parsonsmatt

--- a/esqueleto.cabal
+++ b/esqueleto.cabal
@@ -4,10 +4,10 @@ cabal-version: 1.12
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: c68a1e254e29c461cbf88a8a32c039563f3576cf439b9d947cb3d8dd3916e0e8
+-- hash: b9e04d2f75b2f0eaebcef3906057ae77f42a6455f7bd453c3f7ed399b1669e8e
 
 name:           esqueleto
-version:        2.6.1
+version:        2.6.2
 synopsis:       Type-safe EDSL for SQL queries on persistent backends.
 description:    @esqueleto@ is a bare bones, type-safe EDSL for SQL queries that works with unmodified @persistent@ SQL backends.  Its language closely resembles SQL, so you don't have to learn new concepts, just new syntax, and it's fairly easy to predict the generated SQL and optimize it for your backend. Most kinds of errors committed when writing SQL are caught as compile-time errors---although it is possible to write type-checked @esqueleto@ queries that fail at runtime.
                 .

--- a/package.yaml
+++ b/package.yaml
@@ -1,5 +1,5 @@
 name: esqueleto
-version: 2.6.2
+version: 2.7.0
 synopsis: Type-safe EDSL for SQL queries on persistent backends.
 description: ! '@esqueleto@ is a bare bones, type-safe EDSL for SQL queries
   that works with unmodified @persistent@ SQL backends.  Its

--- a/package.yaml
+++ b/package.yaml
@@ -1,5 +1,5 @@
 name: esqueleto
-version: 2.6.1
+version: 2.6.2
 synopsis: Type-safe EDSL for SQL queries on persistent backends.
 description: ! '@esqueleto@ is a bare bones, type-safe EDSL for SQL queries
   that works with unmodified @persistent@ SQL backends.  Its

--- a/src/Database/Esqueleto/Internal/PersistentImport.hs
+++ b/src/Database/Esqueleto/Internal/PersistentImport.hs
@@ -53,8 +53,6 @@ module Database.Esqueleto.Internal.PersistentImport
   rawQuery,
   rawQueryRes,
   rawSql,
-  executeQQ,
-  sqlQQ,
   askLogFunc,
   close',
   createSqlPool,

